### PR TITLE
Automatic commit: Move 'btp-cf-ext-successfactors' from QA to Production

### PR DESCRIPTION
**Pull request by ekaterina-mitova**
Update step 11 from the tutorial. There is now an automation to configure the subaccount as a trusted service provider in SAP SuccessFactors.

### CODE REVIEW

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please update the tags. "products>" should be "software-product>"

**Comment by ekaterina-mitova** [comment]
Tags are updated.

File: tutorials/btp-cf-ext-s4hanacloud/btp-cf-ext-s4hanacloud.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ title: Extend SAP S/4HANA Cloud on SAP BTP, Cloud Foundry Environment
 description: Extend SAP S/4HANA Cloud with an extension application running on SAP BTP, Cloud Foundry environment using automated integration configuration.
 auto_validation: true
 time: 60
-tags: [ tutorial>intermediate, tutorial>license, products>sap-business-technology-platform, topic>cloud, products>sap-s-4hana-cloud]
-primary_tag: products>sap-btp--cloud-foundry-environment
+tags: [ tutorial>intermediate, tutorial>license, software-product>sap-business-technology-platform, topic>cloud, products>sap-s-4hana-cloud]
```
Comment (MichaelCzcz) [inline tutorials/btp-cf-ext-s4hanacloud/btp-cf-ext-s4hanacloud.md:8]: You have missed one: products>sap-s-4hana-cloud

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by ekaterina-mitova** [comment]
Updated the last tag.

**Pull request merged**